### PR TITLE
Disposing the editor in tree viewer.

### PR DESCRIPTION
**Pull request by Dinesh0723**
Fixes https://github.com/eclipse-pde/eclipse.pde/issues/649

### CODE REVIEW

**Comment by Dinesh0723** [comment]
@gireeshpunathil, could you please verify the changes?

**Comment by gireeshpunathil** [comment]
@Dinesh0723 - sure, will do this in the next 2 days

File: ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/launcher/AbstractPluginBlock.java [hunk 1]
```diff
@@ -1069,21 +1069,10 @@ protected void setVisible(boolean visible) {
...
 	 */
 	protected void refreshTreeView(CheckboxTreeViewer treeView) {
-		// Remove any selection
-		if (treeView.getTree().getItemCount() > 0) {
-			treeView.getTree().setSelection(treeView.getTree().getItem(0));
-		} else {
-			treeView.setSelection(new StructuredSelection(StructuredSelection.EMPTY));
-		}
-
 		// Reset any editors on the tree viewer
```
Comment (gireeshpunathil) [inline ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/launcher/AbstractPluginBlock.java:1076]: @Dinesh0723 - can you explain the reason for this change - removal of the above code? for me, this looks un-necessary, but pls convince me!

**Comment by gireeshpunathil** [comment]
verified that the change set fixes the issue. Have one outstanding question to @Dinesh0723 otherwise we are good to go!

**Comment by Dinesh0723** [comment]
> verified that the change set fixes the issue. Have one outstanding question to @Dinesh0723 otherwise we are good to go!

In an earlier version, the refreshTreeView() was called in the preRefresh() of the viewer of this class
[eclipse.pde/ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/launcher/AbstractPluginBlock.java at 949ef4e561f90c4b21d858084b4bfdb641d8bd38 · eclipse-pde/eclipse.pde · GitHub](https://github.com/eclipse-pde/eclipse.pde/blob/949ef4e561f90c4b21d858084b4bfdb641d8bd38/ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/launcher/AbstractPluginBlock.java). 

This call was removed as part of the bug fix ([Bug 441684 - Improved Plug-in tree in launch configuration · eclipse-pde/eclipse.pde@f9c1123 · GitHub](https://github.com/eclipse-pde/eclipse.pde/commit/f9c11238f01f83ebe90ebf17d74cf24e11c10a6f?diff=split)) and it is not used anywhere, so it is safe to remove setSelection() in the refresh method.

**Review by gireeshpunathil — APPROVED** [review]

**Comment by gireeshpunathil** [comment]
thank you and congratulations @Dinesh0723 for your first PR in PDE! and sorry that you faced issues with the GitHub workflow earlier causing a lot of delay. hope you have a smooth experience now, wishing you good luck for more contributions to this project! 🎉

**Pull request merged**